set_leader_pose:
  leader_pose_topic: /leader_pose
  leader_set_pose_topic: /virtual_leader/set_pose